notebooks/tutorials/04_mis_tutorial.ipynb

```diff
@@ -0,0 +1,1069 @@
+    "    # For example, entrypoint: \"/bin/echo Hello\"\n",
+    "    entrypoint: \"python3 -u /opt/monai/app/app.py\"\n",
+    "\n",
+    "    # Integer value which defines the CPU limit assigned to the MAP container.\n",
+    "    # This value can not be less than 1.\n",
+    "    cpu: 1\n",
+    "\n",
+    "    # Integer value in Megabytes which defines the Memory limit assigned to the MAP container.\n",
+    "    # This value can not be less than 256.\n",
+    "    memory: 7168\n",
+    "\n",
+    "    # Integer value which defines the number of GPUs assigned to the MAP container.\n",
+    "    # This value can not be less than 0.\n",
+    "    gpu: 1\n",
+    "\n",
+    "    # Input directory path of MAP Container.\n",
+    "    # An environment variable `MONAI_INPUTPATH` is mounted in the MAP container\n",
+    "    # with it's value equal to the one provided for this field.\n",
+    "    inputPath: \"/var/monai/input\"\n",
+    "\n",
+    "    # Output directory path of MAP Container.\n",
+    "    # An environment variable `MONAI_OUTPUTPATH` is mounted in the MAP container\n",
+    "    # with it's value equal to the one provided for this field.\n",
+    "    outputPath: \"/var/monai/output\"\n",
+    "\n",
+    "    # Model directory path of MAP Container.\n",
+    "    # For example, modelPath: /opt/monai/models.\n",
+    "    # An environment variable `MONAI_MODELPATH` is mounted in the MAP container\n",
+    "    # with it's value equal to the one provided for this field.\n",
+    "    modelPath: \"/opt/monai/models\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we are ready to deploy our MIS instance:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 27,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "NAME: monai\n",
+      "LAST DEPLOYED: Thu Nov 18 16:17:05 2021\n",
+      "NAMESPACE: default\n",
+      "STATUS: deployed\n",
+      "REVISION: 1\n",
+      "TEST SUITE: None\n"
+     ]
+    }
+   ],
+   "source": [
+    "!helm install monai ./charts/"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 30,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "NAME                                       READY   STATUS    RESTARTS   AGE\n",
+      "monai-inference-service-7fcc7f5b69-qmp9s   1/1     Running   0          76s\n"
+     ]
+    }
+   ],
+   "source": [
+    "!kubectl get po"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Inferencing Segmentation AI with MIS\n",
+    "\n",
+    "With our MIS instance running as a kubernetes pod, we can now submit REST inferencing \n",
+    "requests to it\n",
+    "\n",
+    "We first need to download our input data and compress the dicom files under `/dcm`\n",
```

Comment:
The can mention the earlier chapter as download the test data is done there. If have not done so, then gdown the data.